services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import MovieSession
+
+from django.db.models import QuerySet
+
+from typing import Optional
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: Optional,
+        movie_id: Optional,
+        cinema_hall_id: Optional
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
```

Comment:
It accepts module? 
Because for now you import:
import datetime that mean datetime is module
Do:
`from datetime import datetime`
Or change typing to datetime.datetime